tests/conftest.py

```diff
@@ -12,6 +13,21 @@
 
 from . import utils, TEST_ASSET_GROUP_UUID, TEST_EMPTY_ASSET_GROUP_UUID
 
+# Import all models first for db.relationship to avoid model look up
+# error:
+#
+# sqlalchemy.exc.InvalidRequestError: When initializing mapper
+# mapped class AssetGroupSighting->asset_group_sighting, expression
+# 'Sighting' failed to locate a name ('Sighting'). If this is a
+# class name, consider adding this relationship() to the <class
+# 'app.modules.asset_groups.models.AssetGroupSighting'> class after
+# both dependent classes have been defined.
+project_root = pathlib.Path(__file__).parent.parent
```

Comment:
Any reason you didn't use app.config['ENABLED_MODULES'] here instead or does that not exist on the test side?

Reply:
The app doesn't exist yet at this point.